src/lib/format/tests/TestDecoding.cpp

```diff
@@ -34,7 +34,7 @@ using namespace chip::TLV;
 using namespace chip::TLVMeta;
 using namespace chip::TestData;
 
-const Entry<ItemInfo> _empty_item[0]                 = {};
+const Entry<ItemInfo> _empty_item[]                  = {};
```

Comment:
I think this got fixed in https://github.com/project-chip/connectedhomeip/pull/27973.  Please be careful as you resolve the merge conflict.

Reply:
Reverted.